e2e/tests/cluster_restart.robot

```diff
@@ -15,20 +19,23 @@ ${CONTROL_PLANE_NODE_NETWORK_LATENCY_IN_MS}    0
 
 *** Test Cases ***
 Restart Cluster While Workload Heavy Writing
-    Given Create deployment 0 with rwo volume
-    And Create deployment 1 with rwx volume
-    And Create deployment 2 with rwo and strict-local volume
-    And Create statefulset 0 with rwo volume
-    And Create statefulset 1 with rwx volume
-    And Create statefulset 2 with rwo and strict-local volume
+    Given Create persistentvolumeclaim 0 using RWO volume
+    And Create persistentvolumeclaim 1 using RWX volume
```

Comment:
That's a good question. I'm trying to prevent confusion for the robot with similar keywords.  For instance:
```
And Create statefulset 1 with rwx volume
And Create statefulset 2 with rwo and strict-local volume
```
The robot could see these two as the same keywords.
```
Create statefulset 1 with {rwx} volume
Create statefulset 2 with {rwo and strict-local } volume
```

So we could use "with" when the custom resource (CR) is referenced to something, and "using" otherwise.